test/jdk/com/sun/jdi/JdwpListenTest.java

```diff
@@ -50,7 +50,8 @@ public class JdwpListenTest {
 
     // Set to true to allow testing of attach from wrong address (expected to fail).
     // It's off by default as it causes test time increase and test interference (see JDK-8231915).
-    private static boolean allowNegativeTesting = false;
+    private static boolean allowNegativeTesting =
+        "true".equalsIgnoreCase(System.getProperty("JDI_ALLOW_NEGATIVE_TESTING"));
```

Comment:
That seems a strange name for a system property - it would normally be something like jdk.jdi.allowNegativeTesting. Is this style of naming being used elsewhere?

Reply:
Renamed as suggested.
IIRC we had `-Duse.JTREG_TEST_THREAD_FACTORY=Virtual` some time ago. Other tests use "standard" naming

Reply:
> use.JTREG_TEST_THREAD_FACTORY is not actually a property used by the source. It is just something we add to the mach5 JVM args to show that the test task was done with JTREG_TEST_THREAD_FACTORY=Virtual. It's basically a comment.